README.md

```diff
@@ -16,13 +16,12 @@ helm repo add activemq-artemis https://deviceinsight.github.io/activemq-artemis-
 ```
 
 ## Developer Notes
-
-When you prepare a new release, make sure to build the chart using `./mvnw verify` once the release version has been set.
-Commit the resulting packaged chart `target/helm/*.tgz`.
+When you prepare a new release, make sure to build the chart using ./mvnw verify once the release version has been set. Copy the resulting packaged chart target/helm/*.tgz to charts.
```

Comment:
I could just copy the built chart from the target directory (after mvn verify) and use it to generate the index.yaml but then i don't know how to save it back to the master in the "charts" directory so it is persisted.

As we are not realesing this much often i would just stick to the way w did it with travis (manually adding it)

Reply:
Is there a reason you are not using chart-releaser?

Reply:
After looking into it a bit it doesn't seem too hard. We should get rid of the maven based build and just use the [helm/chart-testing-action](https://github.com/helm/chart-testing-action) and [helm/chart-releaser-action](https://github.com/helm/chart-releaser-action).

I found this example repository too: https://github.com/lippertmarkus/helm-charts

Reply:
@mwalser back to you

Reply:
Looks good :smile: